HW13/properties.ini

```diff
@@ -0,0 +1,5 @@
+[DEFAULT]
+
+url = https://www.saucedemo.com
+username = standard_user
```

Comment:
Properties are good, but it's not a good idea to store credentials in your file on git (especially from your prod or dev envs). Better to provide them as params or for this task just in test file